src/docs/Components/Hero/previews/centered.preview.scss

```diff
@@ -0,0 +1,26 @@
+@import 'settings-tools/_all-settings';
+
+@include import-font-families();
+
+@import 'typography/_t.all-base-typography';
+@import 'components/_c.hero';
+@import 'typography/_t.headings';
```

Comment:
~~@import 'typography/_t.headings';~~ => it's already include in the @import 'typography/_t.all-base-typography';